electronics_lib/JLC_Resistor.py

```diff
@@ -0,0 +1,87 @@
+class JLCPCB_ResistorTable(ESeriesResistor):
+  RESISTANCE = PartsTableColumn(Range)  
+  POWER_DISSIPATION = PartsTableColumn(Range)  
+  PRICE = PartsTableColumn(float)
+  FOOTPRINT = PartsTableColumn(str)
+    
+  @classmethod
+  def _generate_table(cls) -> PartsTable:
+    RESISTOR_MATCHES = {
+    'resistance': "\s?(\d+(\.\d*)?[GMkmunp]?)?[\u03A9]\s",
+    'tolerance': "\s?([\u00B1]\d+(\.\d*)?%)\s",
+    'power_dissipation': "\s?([\u00B1]\d+(\.\d*)?ppm)/℃\s"
```

Comment:
Instead of using `\u00B1` and `\u03A9`, I think you can dump the unicode characters directly in the code?